### PR TITLE
Minor notebook changes

### PR DESCRIPTION
**Pull request by lucyleeow**
Minor working/grammar changes.

### CODE REVIEW

**Review by agramfort — APPROVED** [review]

File: meg_starting_kit.ipynb [hunk 1]
```diff
@@ -53,8 +53,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Brain activity produces electrical currents which are the origin of the magnetic field. Both electric and magnetic signals can be recorded from the scalp of the subject with help of electroencephalography (EEG) and magnetoencephalography (MEG). \n",
-    "Here, we will focus on the magnetic signals recorded by MEG."
+    "Brain activity produces electrical currents which generates a magnetic fields. Both electric and magnetic signals resulting from brain activity can be recorded from the scalp of subjects with help of electroencephalography (EEG) and magnetoencephalography (MEG) respectively.\n",
```
Comment (maikia) [inline meg_starting_kit.ipynb:56]:
```suggestion
    "Brain activity produces electrical currents which generate a magnetic field. Both electric and magnetic signals resulting from brain activity can be recorded from the scalp of subjects with help of electroencephalography (EEG) and magnetoencephalography (MEG) respectively.\n",
```

File: meg_starting_kit.ipynb [hunk 2]
```diff
@@ -68,7 +69,10 @@
...
+    "Communication between brain cells (neurons) happens at synapses. That's where the signal is passed from one neuron to another, causing an electrical current to flow within and outside the neuron. A current flowing into one part of the neuron (forming a current sink), flows within the neuron and must leave it elsewhere (forming a current source). The source and sink pair forms a current dipole. The dipole generated by a single neuron can be therefore understood as a vector with constantly changing direction and length.\n",
+    "\n",
+    "Below you can see a visualization of a positive current (excitator synaptic input) stimulating different locations of 4 neurons. The background color represents the strength of the electric field, the color bar is in nV.\n",
```
Comment (maikia) [inline meg_starting_kit.ipynb:74]:
```suggestion
    "Below you can see a visualization of a positive current (excitatory synaptic input) entering at different locations of 4 neurons. The background color represents the strength of the electric field, the color bar is in nV.\n",
```

File: meg_starting_kit.ipynb [hunk 3]
```diff
@@ -85,7 +89,7 @@
...
+    "        <b>A schema of 4 neurons with the same morphology.</b> Each is stimulated with the synaptic input at different locations (<span style=\"color: #FF0000\">red dot</span>) but at the same time. This is where the current enters the cell. Then, most of it flows through the cell and comes out of the cell body (<span style=\"color: #4B0082\">purple dot</span>) forming a dipole. The direction and relative size of the dipole of each cell is represented by a <span style=\"color: #4B0082\">purple line</span>. Note the differences between the 4 neurons. The colors in the background shows the changes in the extracellular electric field.\n",
```
Comment (maikia) [inline meg_starting_kit.ipynb:92]:
```suggestion
    "        <b>A schema of 4 neurons with the same morphology.</b> Each is stimulated with the synaptic input at different locations (<span style=\"color: #FF0000\">red dot</span>) but at the same time. This is where the current enters the cell. Then, most of it flows through the cell and comes out of the cell body (<span style=\"color: #4B0082\">purple dot</span>) forming a dipole. The direction and relative size of the dipole of each cell is represented by a <span style=\"color: #4B0082\">purple line</span>. Note the differences between the 4 neurons. The colors in the background show the changes in the extracellular electric field.\n",
```

File: meg_starting_kit.ipynb [hunk 4]
```diff
@@ -324,7 +327,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The data has a lot of columns named e1, e2, .. ,e204 and a column named 'subject'. Each column marked with 'e' is a recording from one of the MEG sensors. There are 204 sensors in this MEG recordings. 'subject' is the subject id on whom this recording was performed. Let's see what subjects do we have:"
+    "The data has a lot of columns named e1, e2, ... , e204 and a column named 'subject'. Each column marked with 'e' is the recording from one of the MEG sensors. There are 204 sensors in the MEG recordings. Each row represents the signal from all 204 sensors at one timepoint for one simulation. Recall that each simulation consists of signal generated from one or more sources, which are randomly selected.\n",
+    "\n",
+    "'subject' is the subject id of the particiapnt on whom this recording was performed.\n",
```
Comment (maikia) [inline meg_starting_kit.ipynb:332]:
```suggestion
    "'subject' is the subject id of the participant on whom this recording was performed.\n",
```

File: meg_starting_kit.ipynb [hunk 5]
```diff
@@ -404,14 +418,16 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Before we look at the ground truth let's discuss what do we actually mean by 'source'. The brain is a continous mass and so we could consider millions of points to be a potential source. However, for the sake of this study we subdivided the brain of each subject to 450 regions (225 subregions per hemisphere,  <i>corpus callosum</i> located between the two hemispheres excluded). Each of the subregions is a part of a larger region which has an anatomical meaning (represented by different colors below):"
+    "Before we look at the ground truth let's discuss what we actually mean by 'source'. The brain is a continuous mass and so we could consider millions of points (individual neurons) to be a potential source. However, for the sake of this study we subdivided the brain of each subject into 450 subregions, technical name is 'parcels', (225 parcels per hemisphere, the <i>corpus callosum</i>, located between the two hemispheres, is excluded). Each parcel is a part of a larger region which has an anatomical meaning (represented by different colors below):"
```
Comment (maikia) [inline meg_starting_kit.ipynb:430]:
```suggestion
    "Before we look at the ground truth let's discuss what we actually mean by 'source'. The brain is a continuous mass and so we could consider millions of points (individual neurons) to be a potential source. However, for the sake of this study we subdivided the brain of each subject into 450 subregions, technical name is 'parcels', (225 parcels per hemisphere, the <i>corpus callosum</i>, located between the two hemispheres is excluded). Each parcel is a part of a larger region which has an anatomical meaning (represented by different colors below):"
```

File: meg_starting_kit.ipynb [hunk 6]
```diff
@@ -669,15 +684,7 @@
...
-    "\n",
-    "\n"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "KNeighbors won't accept it in this form. Here, we decide to just drop the whole column and do not use the information about the subjects. We can do it using [ColumnTranformer](https://scikit-learn.org/stable/modules/generated/sklearn.compose.ColumnTransformer.html) or function [make_column_transformer](https://scikit-learn.org/stable/modules/generated/sklearn.compose.make_column_transformer.html):"
+    "KNeighbors won't this data type. Here, we decide to just drop the whole column and do not use the information about the subjects. We can do it using [ColumnTranformer](https://scikit-learn.org/stable/modules/generated/sklearn.compose.ColumnTransformer.html) or function [make_column_transformer](https://scikit-learn.org/stable/modules/generated/sklearn.compose.make_column_transformer.html):"
```
Comment (maikia) [inline meg_starting_kit.ipynb:687]:
```suggestion
    "KNeighbors won't accept this data type. Here, we decide to just drop the whole column and do not use the information about the subjects. We can do it using [ColumnTranformer](https://scikit-learn.org/stable/modules/generated/sklearn.compose.ColumnTransformer.html) or function [make_column_transformer](https://scikit-learn.org/stable/modules/generated/sklearn.compose.make_column_transformer.html):"
```

**Review by maikia — APPROVED** [review]

**Comment by maikia** [comment]
Thx @lucyleeow just few small things

File: meg_starting_kit.ipynb (hunk 5 above)
Comment (lucyleeow) [reply meg_starting_kit.ipynb:430]: I couldnt fine the difference/change here?

Comment (lucyleeow) [reply meg_starting_kit.ipynb:430]: oh wait i see the comma

**Comment by maikia** [comment]
thx @lucyleeow

**Pull request merged**